packages/strapi-admin/admin/src/components/Logout/index.js

```diff
@@ -8,32 +8,27 @@
 import React, { useState } from 'react';
 import { FormattedMessage } from 'react-intl';
 import { withRouter } from 'react-router-dom';
-import { get } from 'lodash';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { ButtonDropdown, DropdownItem, DropdownMenu, DropdownToggle } from 'reactstrap';
 import { auth } from 'strapi-helper-plugin';
 import Wrapper from './components';
 
 const Logout = ({ history: { push } }) => {
   const [isOpen, setIsOpen] = useState(false);
-  const toggle = () => setIsOpen(prev => !prev);
-  const handleGoTo = () => {
-    const id = get(auth.getUserInfo(), 'id');
 
+  const handleGoTo = () => {
```

Comment:
The name `handleGoTo` looks like you can pass a param to go to a specific page.
It should be `handleGoToMe` or `handleGoToProfile`.